quil-rs/src/parser/token.rs

```diff
@@ -67,6 +67,78 @@ where
     }
 }
 
+/// The subset of [`Token`]s which (a) do not contain more specific data and (b) are keywords.  Used
```

Comment:
what does `more specific data` mean here? Like arguments?

Reply:
Yes, that’s a better way to say it, thanks!